policies/recipes/steps.go

```diff
@@ -499,5 +500,14 @@ func executeCommand(cmd string, args []string, workDir string, runEnvs []string,
 			}
 		}
 	}
+	logger.Infof("Combined output for %q command:\n%s", cmd, o)
```

Comment:
I added some debugging here because I found it incredibly useful while trying to debug some of these issues, let me know if this shouldn't be here.